.precaution.ymal.example

```diff
@@ -0,0 +1,13 @@
+# Copyright 2018 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+
+# That way you can say which security linters do you want to skip
+skip_linters: 'bandit'
```

Comment:
This makes more sense when we have a plugin model for the linters.  Right now, we have just gosec and bandit which are hardcoded.